lib/Sema/TypeCheckStorage.cpp

```diff
@@ -3225,7 +3225,7 @@ PropertyWrapperInitializerInfoRequest::evaluate(Evaluator &evaluator,
       // Check initializer effects.
       auto *initContext = new (ctx) PropertyWrapperInitializer(
           dc, param, PropertyWrapperInitializer::Kind::ProjectedValue);
-      checkInitializerActorIsolation(initContext, projectedValueInit);
+      (void)projection->getInitializerIsolation();
```

Comment:
Just like in the other case I mentioned - might be worth replacing this direct call with a top-level `computeRequiredIsolation`

Reply:
I removed `swift::computeRequiredIsolation` in favor of always going through the request via `getInitializerIsolation`

Reply:
Sounds good!